plugin/path_role.go

```diff
@@ -217,6 +236,11 @@ func pathsRole(b *GcpAuthBackend) []*framework.Path {
 		// Edit labels on an GCE role
 		{
 			Pattern: fmt.Sprintf("role/%s/labels", framework.GenericNameRegex("name")),
+			DisplayAttrs: &framework.DisplayAttributes{
+				OperationPrefix: operationPrefixGoogleCloud,
+				OperationVerb:   "edit",
```

Comment:
This seems like a new OperationVerb, should this be `update` for consistency?

Reply:
This operation edits the labels with "add" / "remove" arrays. The name is also from the [docs](https://developer.hashicorp.com/vault/api-docs/auth/gcp#edit-labels-on-gce-role)